src/Features/Core/Portable/EmbeddedLanguages/IEmbeddedLanguageFeatures.cs

```diff
@@ -27,6 +26,6 @@ internal interface IEmbeddedLanguageFeatures : IEmbeddedLanguage
         /// individual providers and expose them as one single completion provider to
         /// the rest of Roslyn.
         /// </summary>
-        EmbeddedLanguageCompletionProvider CompletionProvider { get; }
+        EmbeddedLanguageCompletionProvider? CompletionProvider { get; }
```

Comment:
nullable now because json doesn't support any special completion behavior.

Reply:
You don't want to implement JSON schema support? :smile:

Reply:
keelmee